tests/orquestra/integrations/qiskit/backend/backend_test.py

```diff
@@ -139,10 +139,12 @@ def test_aggregate_measurements_extract_correct_qubits(
         self, backend_with_readout_correction
     ):
         # Given
-        pickling_off = open("jobs_and_batches_with_different_qubits.pickle", "rb")
-        jobs = pickle.load(pickling_off)
-        batches = pickle.load(pickling_off)
-        pickling_off.close()
+        extract_pickle_objects = open(
```

Comment:
This is probably just me being picky, but "extract" is a verb, and this is about "objects", so noun.
Shouldn't it be rather `extracted_pickle_objects`?